packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -1237,21 +1235,22 @@ export function extractGdocPageData(gdoc: OwidGdoc) {
                 ]),
             }
         })
-        .with({ type: OwidGdocType.DataInsight }, () => {
-            const dataInsightGdoc = gdoc as OwidGdocDataInsightInterface
+        .when(isDataInsightGdoc, (dataInsightGdoc) => {
             return {
                 ...commonProps,
                 ...R.pick(dataInsightGdoc, ["latestDataInsights"]),
             }
         })
-        .with({ type: OwidGdocType.Author }, () => {
-            const authorGdoc = gdoc as OwidGdocAuthorInterface
+        .when(isAuthorGdoc, (authorGdoc) => {
             return {
                 ...commonProps,
                 ...R.pick(authorGdoc, ["latestWorkLinks"]),
             }
         })
-        .otherwise(() => commonProps)
+        .otherwise(() => ({
+            ...commonProps,
+            content: gdoc.content,
```

Comment:
that I'm less excited about. It does technically remove the need to assert in `deserialize..` but I think that if we are in the business of asserting, then I'd rather do it explicitly.

I would remove it if you're thinking of merging.